AllReadyApp/Web-App/AllReady/Views/Campaign/ManageCampaign.cshtml

```diff
@@ -0,0 +1,102 @@
+﻿@model IList<AllReady.ViewModels.Campaign.CampaignViewModel>
+@using AllReady.Security
+@using AllReady.ViewModels.Campaign
+
```

Comment:
remove the two using lines above this comment. These are not being used on this page